CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1201,7 +1201,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Other settings for more _particular_ users." xml:space="preserve">
         <source>Other settings for more _particular_ users.</source>
-        <target state="translated">Weitere Einstellungen für _besondere_ Nutzende.</target>
+        <target state="translated">Weitere Einstellungen für _wählerische_ Nutzende.</target>
```

Comment:
"besondere" means "special", "wählerische" means "picky". I think your intention was to go with the latter.